opta/flyte.yaml

```
@@ -43,7 +43,9 @@ modules:
     links:
       - s3: [ "write" ]
   - type: helm-chart
-    chart: "../helm" # NOTE: relative path to chart
+    repository: https://flyteorg.github.io/flyte
+    chart: flyte
+    chart_version: "0.17.3-b2"
```

I think we may keep this PR open until we release the chart... with all the changes we need... ?
or else, you can pin it to the one currently "released":
```suggestion
    chart_version: "v0.17.2"
```

And we can open a new PR when a new one is released...

In fact, @evalsocket, can we have this line get updated in a github WF?
When a release is created, a github WF can get triggered to update this file and open a PR for us to merge... ?

---

Yes but why can't we use latest tag for deployment. Helm will install latest if version is not specified